Bottom: ac0cd1105f954e9e50a7c180f4cc60b9f5e38edc
Top:    f25deae9304eabb39b51a821f598a97f2e18f458
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-16 23:51:41 +0000

Adjust STEP export to 0.2mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 43222ef..e855b45 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -63,7 +63,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(0.2)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
 
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 7ac041c..d413f71 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -45,7 +45,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(0.2)
 
 
 HID step_hid;
